extension/popup.js

```diff
@@ -60,13 +58,14 @@ function initUI(items) {
 
     var a = document.createElement('a');
     a.href = '#';
+	var BASE_URL = localStorage['api_endpoint'];
     a.addEventListener('click', function(e) {
-        chrome.tabs.update(null, {
-            url: BASE_URL + '/' + change._number
-          });
+      chrome.tabs.update(null, {
+          url: BASE_URL + '/' + this.textContent
       });
+    });
 
-    a.textContent = 'cl/' + change._number;
```

Comment:
Please keep 'cl/' this is a part of another feature I'm intending to add, sorry for not writing up the issue before, see #18.

Instead why don't we do:
```javascript
a.addEventListener('click', function(e) {
  chrome.tabs.update(null, {
    url: BASE_URL + '/' + this.getAttribute('change-id');
  });
})
a.setAttribute('change-id', change._number)
```